src/Command/CommandBase.php

```diff
@@ -1170,16 +1170,16 @@ protected function checkForNewVersion(InputInterface $input, OutputInterface $ou
     if (strpos($input->getArgument('command'), 'api:') !== FALSE) {
       return;
     }
-    // Bale for development builds.
+    // Bail for development builds.
     if ($this->getApplication()->getVersion() == '@package_version@') {
       return;
     }
-    // Bale in Cloud IDEs to avoid hitting Github API rate limits.
-    if (!AcquiaDrupalEnvironmentDetector::isAhIdeEnv()) {
+    // Bail in Cloud IDEs to avoid hitting Github API rate limits.
+    if (AcquiaDrupalEnvironmentDetector::isAhIdeEnv()) {
```

Comment:
We don't have to solve this here, but I don't like that ACLI has hardcoded support for IDEs. I'd rather support an environment variable like `ACLI_DISABLE_UPDATES`, which could be set in any environment.

Reply:
ACLI has hardcoded support for IDEs in _a lot_ of places. Here we're just fixing a but, not integrating it more deeply.